WcaOnRails/app/models/abstract_cached_model.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+class AbstractCachedModel < ActiveRecord::Base
```

Comment:
I think it should be a [Rails concern](http://api.rubyonrails.org/classes/ActiveSupport/Concern.html). [There](https://github.com/cubing/worldcubeassociation.org/tree/master/WcaOnRails/app/models/concerns) is even a directory for them already =)
It would be much nicer to just do
```ruby
MyAwesomeModel < ActiveRecord::Base
  include Cachable
end
```